G2P.pm

```diff
@@ -70,19 +70,16 @@ limitations under the License.
                          allele frequencies. Allele frequencies are retrieved from VCF files. If
                          af_from_vcf is set to 1 but no VCF collections are specified with '--af_from_vcf_keys'
                          all available VCF collections are included. 
-                         Available VCF collections: 'topmed', 'uk10k', 'gnomADe', 'gnomADe_r2.1.1', 'gnomADg', 'gnomADg_v3.1.2'.
+                         Available VCF collections: 'topmed', 'uk10k', 'gnomADe', 'gnomADe_r2.1.1', 'gnomADg', 'gnomADg_v3.1.2', 'gnomADev4.1', 'gnomADgv4.1'.
                          Separate multiple values with '&'.
                          VCF collections contain the following populations: 
                          * 'topmed' - TOPMed (available for GRCh37 and GRCh38).
                          * 'uk10k' - ALSPAC, TWINSUK (available for GRCh37 and GRCh38).
-                         * 'gnomADe' & 'gnomADe_r2.1.1' - gnomADe:AFR, gnomADe:ALL, gnomADe:AMR, gnomADe:ASJ, gnomADe:EAS, gnomADe:FIN, gnomADe:NFE, gnomADe:OTH, gnomADe:SAS (for GRCh37 and GRCh38 respectively).
-                         * 'gnomADg' & 'gnomADg_v3.1.2' - gnomADg:AFR, gnomADg:ALL, gnomADg:AMR, gnomADg:ASJ, gnomADg:EAS, gnomADg:FIN, gnomADg:NFE, gnomADg:OTH (for GRCh37 and GRCh38 respectively).
+                         * 'gnomADe' & 'gnomADe_r2.1.1 & 'gnomADev4.1' - gnomADe:AFR, gnomADe:ALL, gnomADe:AMR, gnomADe:ASJ, gnomADe:EAS, gnomADe:FIN, gnomADe:NFE, gnomADe:OTH, gnomADe:SAS (for GRCh37 and GRCh38 respectively).
+                         * 'gnomADg' & 'gnomADg_v3.1.2' & 'gnomADgv4.1' - gnomADg:AFR, gnomADg:ALL, gnomADg:AMR, gnomADg:ASJ, gnomADg:EAS, gnomADg:FIN, gnomADg:NFE, gnomADg:OTH (for GRCh37 and GRCh38 respectively).
                          Need to use 'af_from_vcf' parameter to use this option. 
- default_af            : default frequency of the input variant if no frequency data is
```

Comment:
The option 'default_af' was not removed in https://github.com/Ensembl/VEP_plugins/pull/770 or https://github.com/Ensembl/VEP_plugins/pull/777
As this PR is only going to be available for release/113 there is no point to remove the option here.

Reply:
I added this because I wanted us to remove it from the Perldoc, I will also make a mini PR for 114, thats the plan.

Reply:
Yes, the plan is for future releases.